yt_dlp/extractor/tenplay.py

```diff
@@ -102,14 +109,19 @@ def _real_extract(self, url):
         video_data = self._download_json(
             f'https://vod.ten.com.au/api/videos/bcquery?command=find_videos_by_id&video_id={data["altId"]}',
             content_id, 'Downloading video JSON')
+        # Dash URL 403s, changing the m3u8 format works
```

Comment:
For next clean-up:
```suggestion
        # Dash URL 404s, changing the m3u8 format works
```
Unless ofc 403 **is** being seen...